.github/workflows/test_frontend_part2.yaml

```diff
@@ -0,0 +1,162 @@
+name: Test Skyportal Front-end Part 2
```

Comment:
Instead of duplicating the file, it should be easier to define a [matrix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).

Reply:
I'll look into that!